kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigWriter.java

```diff
@@ -97,8 +97,7 @@ public void generateTempWpaSupplicantConf(WifiConfig wifiConfig, String interfac
     public void generateTempWpaSupplicantConf() throws KuraException {
 
         try {
-            String fileAsString = readResource(
-                    "/src/main/resources/wifi/wpasupplicant.conf");
+            String fileAsString = readResource("/src/main/resources/wifi/wpasupplicant.conf");
```

Comment:
Maybe this file path should be extracted to a constant.

Reply:
done